docs/source/concepts/ledger-model/index.rst

```diff
@@ -3,8 +3,8 @@
 
 .. _da-ledgers:
 
-Daml Ledger Model
-#################
+Understanding the Daml Ledger Model
+###################################
```

Comment:
```suggestion
Daml Ledger Model
#################
```
I don't think it's necessary to add "Understanding the" here. It feels more natural and simple. What is this page about? The "Daml Ledger Model", that's what it says in the title.